lib/utils/extensions.dart

```diff
@@ -19,4 +18,9 @@ extension TinyColorExtension on Color {
   }
 
   Color smallTiny() => this.tiny(3);
+}
+
+extension CustomCardTheme on CardTheme {
+  Color get titleLighterBackgroundColor => Color(0xff373f5c);
```

Comment:
You can also define an extension method `titleColor(BuildContext context)` which returns the appropriate color:
```
Color titleColor(BuildContext context) => TinyColor(Theme.of(context).cardTheme.color).isDark()
                  ? TinyColor(Theme.of(context).cardTheme.color).lighten(3).color	                  ? Theme.of(context).cardTheme.titleLighterBackgroundColor
                  : TinyColor(Theme.of(context).cardTheme.color).darken(3).color,	                  : Theme.of(context).cardTheme.titleDarkerBackgroundColor,
```

This would avoid to repeat the test everywhere the card's title background color is used

Reply:
It's done !